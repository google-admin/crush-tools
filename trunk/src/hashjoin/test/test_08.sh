test_number=08
description="join by index with header-only input"

infile="$test_dir/input_header.log"
dimfile="$test_dir/dimension_header.log"
outfile="$test_dir/test_$test_number.actual"
expected="$test_dir/test_$test_number.expected"

head -1 $infile | $bin -k 1,2 -l 1,2 -j 3,4 -f $dimfile \
  > "$outfile"

if [ $? -ne 0 ] ||
   [ "`diff -q $outfile $expected`" ]; then
  test_status $test_number 1 "$description" FAIL
else
  test_status $test_number 1 "$description" PASS
  rm "$outfile"
fi
