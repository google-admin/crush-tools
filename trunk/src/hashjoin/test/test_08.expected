Field-0,Field-1,Something-Else,Field-2,Field-3
